.github/workflows/ci.yml

```diff
@@ -23,13 +23,21 @@ jobs:
         with:
           node-version: ${{ matrix.node-version }}
 
+      - name: "Create env file"
+        run: |
+          touch .env
+          echo POLYGON_PRIVATE_KEY=${{ secrets.POLYGON_PRIVATE_KEY }} >> .env
+          echo POLYGON_URL=${{ secrets.POLYGON_URL }} >> .env
+          cat .env
```

Comment:
I think you can remove `cat` and `touch` above.

Reply:
Still doesn't work anyway 🤷🏻‍♂️